src/include/daos_srv/bio.h

```diff
@@ -717,10 +717,10 @@ void *bio_iod_bulk(struct bio_desc *biod, int sgl_idx, int iov_idx,
  * Wrapper of ABT_thread_yield()
  */
 static inline void
-bio_yield(void)
+bio_yield(struct umem_instance *umm)
 {
 #ifdef DAOS_PMEM_BUILD
-	D_ASSERT(umem_tx_none());
+	D_ASSERT(umem_tx_none(umm));
```

Comment:
This assert isn't true anymore, and so far I don't have a good idea to fix it, so I'm afraid that we might have to get rid of this assert at the end? If so, I'm wondering if it's necessary to introduce so many changes to pass in 'umm' in this PR, and remove them in following PR?

Reply:
why "umem_tx_none(umm)" is not true here? for admem it means no TX in TLS seems it still is true?
and bio_yield() need to check if it is able to yield, so still need that kind of checking?

Reply:
yes, I think you are right.